### ISSUE TITLE
There is a list of existing PlaceCal instances - with links to them

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#1818]
## Acceptance Criteria

- [ ] #1844
- [ ] #1845
- [ ] The site has a list of current PlaceCal instances
- [ ] TBC: Each "teaser" contains the name of the site, the link to the site, MORE?
- [ ] Trans Dimension is on home page (new ticket)
- [ ] Find your PlaceCal page has all
- [ ] Area component is on home & find PlaceCal page

## Blocked by
- [x] Kim make a list of live PlaceCal sites
- [ ] Mark wireframe what the teaser might look like
- [ ] FInd out what happens if we publish a community interest site

## implementation plan

This is now the "find your PlaceCal" design

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/70dcb599-54a4-49b6-bdda-af53adc6f6b0)

only the bottom element appears on the homepage

**Comment by katjam** [comment]
![image](https://user-images.githubusercontent.com/861172/235635255-04fda289-5110-4b7c-9cf9-1fe75afd879a.png)

**Comment by kimadactyl** [comment]
**Groups**

Conscious Collective Manchester
Flourish Together
GM Systems Changers
The Trans DImension

**Areas**

Manchester
Oldham
Norwich
Torbay
London
Christchurch

**To get back to / leave up for now:**

OL1 Oldham
Marvellous Mossley

**Comment by studiosquid** [comment]
Cropped wireframe of the new components needed for this, I've suggested two types might be useful one where we include description, some where we don't. I think this marries up quite well to Partnerships + Neighbourhoods.

Mark not designed in detail yet but similar / comparable components were mocked up in the Instance improvements where we list four elements side by side. 

![PlaceCal_2023_Wires_4May_crop](https://user-images.githubusercontent.com/9461465/236183310-a6c13ad2-4657-4d18-b8e6-4ceccc5abd63.png)

Have attached a flat of the instance so you can see what I'm referencing. 

![1_Web 1366 – Instance - Portal](https://user-images.githubusercontent.com/9461465/236183192-55259435-9f40-498e-acdf-425f9f6454e8.png)

**Comment by studiosquid** [comment]
Have done a very crude mock of how these new components could look based on existing elements. 

![Draftcomponents](https://user-images.githubusercontent.com/9461465/236193588-95d1b0c8-f4b0-4d3c-a924-28ec3c57ef52.png)

**Comment by katjam** [comment]
Closing as it covered by #1844 and #1845

**Comment by katjam** [comment]
Verified on staging:
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/861172/ae297df8-bbb1-4911-be71-56c50870bb19)
